### DISCUSSION TITLE
Unable to configure domains

### DISCUSSION THREAD

**Discussion by KrofDrakula** [vercel/community#5625] [Bug Report]
### Page to Investigate

https://vercel.com/dashboard/domains/lk-apolon.si

### Steps to Reproduce

Opening any domain registered results in a 404 page. I cannot configure my DNS settings for any of the domains I'm registered with.

![image](https://github.com/vercel/community/assets/112161/4d11b694-1b6b-4c97-9dad-d74293ad5bb7)

**Comment by mikepsinn** [comment]
#METOO

**Comment by b0yw0nder3100** [comment]
I'm having the same issue as well

**Comment by mikepsinn** [comment]
It's a URL issue.  You can manually open the domain page by typing in the URL with this format:

https://vercel.com/{vercel_username}/~/domains/{domain}

**Comment by KrofDrakula** [comment]
@mikepsinn Sure, but that is the link the components link to from the list of domains. It's still a bug.

**Reply by mikepsinn** [reply]
Right, I was just saying how people can manage their domain until it's fixed.

**Comment by amyegan** [comment]
Thanks for reporting this. We're working to fix it now.

And thank you @mikepsinn for sharing your workaround https://github.com/vercel/community/discussions/5625#discussioncomment-8261722

**Reply by amyegan** [reply]
This should be working normally now. Please let me know if you still have trouble with it! 🙏
